config/19.1/ports.conf

```diff
@@ -210,6 +210,8 @@ sysutils/munin-node				arm,arm64,i386
 sysutils/node_exporter				arm,arm64
 sysutils/nut					arm,arm64
 sysutils/pftop
+sysutils/py-psutil@py${PRODUCT_PYTHON2}
+sysutils/py-psutil@py${PRODUCT_PYTHON3}
```

Comment:
you can remove that one - or even better remove the other one and port the file to 3